### ISSUE TITLE
There is no new movies for the last 2 months.. 

### ISSUE THREAD

**Issue by zorvalth** [sjlu/popular-movies#21]
The last movie in the list is from 2 months ago, is there something broken with the script?

**Comment by dhaos21** [comment]
Yeah I have a feeling it's due to the coronavirus and all the cinema's being closed, probably missed out on quite a few released that went "straight to VOD"

**Comment by sjlu** [comment]
@dhaos21, you are correct. We use Metacritic's in-theaters as our seed. Unfortunately, because of COVID-19 there have been no new movies in theaters recently.

See: https://www.metacritic.com/browse/movies/release-date/theaters/date?ttype[]=1

**Comment by dhaos21** [comment]
Is there anyway of integrating Amazon/Netflix into this list as there are more and more Amazon/Netflix originals coming out that would not appear on the Metacritic in-theatre seed.

**Comment by codsane** [comment]
> 
> 
> Yes it will be great to get any flow of popular movies..

If your sole purpose for this movie list was import into one of the *arr variants, then I highly recommend something like [traktarr](https://github.com/l3uddz/traktarr).

It pulls from any trakt.tv lists, and can be setup to run on a schedule or just use it manually via the CLI.

**Comment by dhaos21** [comment]
Well there are normal trakt lists in the arrs as well box office etc but
they are all in the same boat atm with there being no cinemas open.

**Comment by codsane** [comment]
> Well there are normal trakt lists in the arrs as well box office etc but they are all in the same boat atm with there being no cinemas open.

IIRC I don't believe Trakt suffers from the same problem; as it is a "scrobbler". Feeds are curated from what users are currently watching (Trakt has integrations for Plex, and many other HTPC solutions which log playback).

Traktarr just provides much more control over what to you want to add to Radarr from Trakt. For example you could use this to add the 20 most recently released movies from Trakt's trending list:

`traktarr movies -t trending -s release -l 20`
This tells Traktarr to grab all the trending movies from Trakt, sort them by release date, and then add a maximum of 20 of them to Radarr.

**Comment by dhaos21** [comment]
I did have a play with it a couple of months ago but it was adding really
obscure stuff but I did leave most the settings at default ie rating etc.
But yeah the arrs can add lists of what people are watching now over the
last 7 days via the lists screen.
